src/styles/variables.scss

```diff
@@ -0,0 +1,13 @@
+$width-boxs: 100px; // Ширина одного дня
```

Comment:
The variable name `$width-boxs` seems to be a typo and does not follow the naming convention. Consider renaming it to `$box-width` to better reflect its purpose and to match the singular form as the variable represents the width of a single box.